lib/definition/component-info.ts

```diff
@@ -93,3 +106,12 @@ function addRestrictChildrenInfo(
     }
     field.restrictChildren = allowedComponentsNames;
 }
+
+function parseProperties(properties: ComponentProperty[]) {
+    return properties.reduce((infoProperties, property) => {
+        if (property.name) {
```

Comment:
I added this condition to rule out header properties, stumbled upon this by accident as there was a header property in a slideshow component of one of the tests. Not really sure if this is the best way to check it though? I was also wondering if it's correct that the component property `name` is currently [not optional](https://github.com/WoodWing/studio-component-set-tools/blob/master/lib/models/components-definition.ts#L136)?

Reply:
`header` indeed ended up being weird. Maybe check on the `control.type`? According to the json schema it is not optional, but `header` bypasses it because it is merged in...
It's an ugly exception.